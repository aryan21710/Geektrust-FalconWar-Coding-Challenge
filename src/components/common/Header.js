import React from 'react';
import { Heading, HeaderWrapper } from './StyledComponents/StyledCommonComp';

const Header = () => {
	return (
		<HeaderWrapper>
			<Heading>WELCOME TO THE BATTLE OF TARA B</Heading>
		</HeaderWrapper>
	);
};

export default Header;
