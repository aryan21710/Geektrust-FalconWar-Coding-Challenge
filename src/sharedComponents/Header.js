import React from 'react';
import { Heading, HeaderWrapper } from './styles';

const Header = () => {
	return (
		<HeaderWrapper>
			<Heading data-testid="Headerh1">WELCOME TO THE BATTLE OF TARA B</Heading>
		</HeaderWrapper>
	);
};

export default Header;
