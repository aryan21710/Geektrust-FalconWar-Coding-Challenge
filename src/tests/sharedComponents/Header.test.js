import React from 'react';
import Header from '../../sharedComponents/Header';
import { shallow, mount, render } from '../../../enzyme';
import toJSON from  'enzyme-to-json';

describe('TESTING HEADER COMPONENT', () => {
	test('SNAPSHOT', () => {
		const wrapper = shallow(<Header />);
		expect(toJSON(wrapper)).toMatchSnapshot();
	});

	test('renders text properly', () => {
        const wrapper = mount(<Header/>)
        expect(wrapper.find('h1').length).toBe(1);
        expect(wrapper.find('h1').text()).toBe('WELCOME TO THE BATTLE OF TARA B');

	});
});
