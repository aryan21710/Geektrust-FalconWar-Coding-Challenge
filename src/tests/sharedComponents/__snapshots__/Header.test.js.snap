// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TESTING HEADER COMPONENT SNAPSHOT 1`] = `
<styled.div>
  <styled.h1
    data-testid="Headerh1"
  >
    WELCOME TO THE BATTLE OF TARA B
  </styled.h1>
</styled.div>
`;
